src/components/Order/Order.jsx

```diff
@@ -0,0 +1,17 @@
+import RestaurantButton from "../RestaurantButton/RestaurantButton";
+import React, { useState } from "react";
+import "./Order.scss";
+
+export default function Order({ orderType }) {
+  const [orders, setOrders] = useState(0);
```

Comment:
We always create the SetState var to be camel case. So your `setorder` should be camel case. Can you change that please?